app/assets/javascripts/historical.js.es6

```diff
@@ -0,0 +1,176 @@
+class HistoricalChart {
+  constructor(data, chartElement) {
+    this.data = data;
+    this.chartElement = chartElement;
+    this.dimensions = {};
+    this.filters = {};
+    this.counts = [];
+  }
+
+  filtersUpdated(data) {
+    this.data = data;
+    this.update();
+  }
+
+  render() {
+
+    function setDates() {
+      var latestDate = new Date();
+      var counterDate = new Date(2016, 1, 1);
+      var week;
+      var dates = [counterDate];
+
+      while (latestDate >= counterDate) {
+        week = counterDate.getDate() + 7
+        counterDate = new Date(counterDate.setDate(++week));
+        dates.push(counterDate);
+      }
+      return dates;
+    }
+
+    var datesArray = setDates();
+
+    var bookings = crossfilter(this.data);
+
+    function getCounts() {
+      var counts = [];
+      datesArray.forEach(function (date) {
+        let currentDate = date;
+        var bookingDateTime = bookings.dimension(d => d.booking_date_time);
+        var releaseDateTime = bookings.dimension(d => d.release_date_time);
+        bookingDateTime.filterFunction(d => d < currentDate);
+        releaseDateTime.filterFunction(d => d == null || d > currentDate);
+        counts.push(bookings.groupAll().reduceCount().value());
+        bookingDateTime.dispose()
+        releaseDateTime.dispose()
+      });
+      return counts;
+    }
+    this.counts = getCounts()
+
+    var data = [{
+      date: datesArray,
+      count: this.counts
+    }];
+
+    var xy_chart = d3_xy_chart()
+
+    var svg = d3.select('.historical').append('svg')
+      .datum(data)
+      .call(xy_chart);
+
+    function d3_xy_chart() {
+      var width = document.getElementsByClassName('historical')[0].getBoundingClientRect().width,
+        height = 480;
+
+      function chart(selection) {
+        selection.each(function (datasets) {
+
+          var margin = {
+              top: 20,
+              right: 30,
+              bottom: 30,
+              left: 30
+            },
+            innerwidth = width - margin.left - margin.right,
+            innerheight = height - margin.top - margin.bottom;
+
+          var x_scale = d3.scaleTime()
+            .range([0, innerwidth])
+            .domain([d3.min(datasets, function (d) {
+                return d3.min(d.date);
+              }),
+              d3.max(datasets, function (d) {
+                return d3.max(d.date);
+              })
+            ]);
+
+          var y_scale = d3.scaleLinear()
+            .range([innerheight, 0])
+            .domain([d3.min(datasets, function (d) {
```

Comment:
Can we set the lower end of this domain to `0`, so that the bottom of our chart represents 0 people? I think it might give a more accurate picture of the population trends.